docs/tutorials/transformation-and-normalization/transformations-with-airbyte.md

```diff
@@ -0,0 +1,41 @@
+# Transformations with Airbyte \(Part 3/3\)
+
+## Overview
+
+This tutorial will describe how to push a custom dbt transformation project back to Airbyte to use during syncs.
+
+This tutorial is the last part of the tutorial serie on transformations after [Connecting EL with T using SQL](transformations-with-sql.md) and 
+[connecting EL with T using DBT](transformations-with-dbt.md).
+
+(Example outputs are updated with Airbyte version 0.23.0-alpha from May 2021)
+
+## Transformations with Airbyte
+
+After replication of data from a source connector (Extract) to a destination connector (Load), multiple optional transformations steps can now be applied as part of an Airbyte Sync. Possible transformations are:
+
+1. Basic normalization transformations as automatically generated by Airbyte dbt code generator.
+2. Customized normalization transformations as edited by the user (and disable the default generated normalization one)
+3. Customized business transformations as specified by the user.
+
+Let's connect my mono-repo Business Intelligence project stored in a private github git repository to update the related tables and dashboards when my Airbyte syncs completes.
+
+Note that if you need to connect to a private git repository, one of the recommended way is to generate `Private Access Tokens` that can be used instead of a password. Then, you'd be able to include the credentials in the git repository url:
+
+- [GitHub - Personal Access Tokens](https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token)
+- [Gitlab - Personal Access Tokens](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html)
+
+And then use it for cloning:
+
+    git clone https://username:token@github.com/user/repo
+
+Where `https://username:token@github.com/user/repo` is the git repository url.
+
+Since I am using a mono-repo from my organisation and other people/team/department may also contribute their dbt models in this centralized location too, we may end up with many dbt models and sources to build our complete data warehouse...
```

Comment:
I know `organisation` is correct in British English, I apologize for my instinct to make this American.

```suggestion
Since I am using a mono-repo from my organization, other team members or departments may also contribute their dbt models to this centralized location. This will give us many dbt models and sources to build our complete data warehouse...
```

Reply:
No problem, it's better!

My English is all over the place...